media/transcoder/create_job_with_embedded_captions.go

```diff
@@ -0,0 +1,157 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package transcoder
+
+// [START transcoder_create_job_with_embedded_captions]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	transcoder "cloud.google.com/go/video/transcoder/apiv1"
+	transcoderpb "google.golang.org/genproto/googleapis/cloud/video/transcoder/v1"
+)
+
+// createJobWithEmbeddedCaptions creates a job that embeds captions in the
+// output video. See https://cloud.google.com/transcoder/docs/how-to/captions-and-subtitles
+// for more information.
+func createJobWithEmbeddedCaptions(w io.Writer, projectID string, location string, inputVideoURI string, inputCaptionsURI string, outputURI string) error {
+	// projectID := "my-project-id"
+	// location := "us-central1"
+	// inputVideoURI := "gs://my-bucket/my-video-file"
+	// inputCaptionsURI := "gs://my-bucket/my-captions-file"
+	// outputURI := "gs://my-bucket/my-output-folder/"
+
+	ctx := context.Background()
+	client, err := transcoder.NewClient(ctx)
+	if err != nil {
+		return fmt.Errorf("NewClient: %v", err)
+	}
+	defer client.Close()
+
+	req := &transcoderpb.CreateJobRequest{
+		Parent: fmt.Sprintf("projects/%s/locations/%s", projectID, location),
+		Job: &transcoderpb.Job{
+			OutputUri: outputURI,
+			JobConfig: &transcoderpb.Job_Config{
+				Config: &transcoderpb.JobConfig{
+					Inputs: []*transcoderpb.Input{
+						&transcoderpb.Input{
```

Comment:
When instantiating a slice item you can omit the types of the individual items. So here you can remove the `&transcoderpb.Input` part. There are about a dozen of these in this file.

Reply:
Also in other files.

Reply:
Done in these files. Filed b/218557754 to clean up the rest of the files.